drone/exec/exec.go

```diff
@@ -148,6 +150,17 @@ func exec(c *cli.Context) error {
 		return err
 	}
 
+	// workaround to allow yaml anchors
+	var anyJSON map[string]interface{}
+	err = yamlv2.Unmarshal([]byte(data), &anyJSON)
+	if err != nil {
+		return err
+	}
+	data, err = yamlv2.Marshal(anyJSON)
+	if err != nil {
+		return err
+	}
```

Comment:
0.9 supports multi-document yaml files which would be problematic here. But we can definitely backport this to 0.8.  I will create an 0.8 branch tomorrow morning that we can use.

Reply:
Is there documentation around that so I can try to understand the issue.  Looking at the cli source it seems it looks for a single file, defaulting to `.drone.yml`, and reads the file and starts using it.

Thanks for the help on this!

Reply:
yes, the `yaml.Unmarshal` will not work with a multi-document yaml like this https://github.com/drone/drone-git/blob/master/.drone.yml :)